Assignment/solution_student/css/style.css

```diff
@@ -34,8 +36,251 @@ body {
   color: black;
   font-size: 16px;
   background: white;
+  margin: 0 auto;
+  width: 70%;
+  /*    line-height: 1.5; //// I don't think I need this */
```

Comment:
It's not mandatory, but what it does do is establish a base line-height for all of your paragraphs so they have some vertical spacing :)